debugtools/DDR_VM/src/com/ibm/j9ddr/tools/PointerGenerator.java

```diff
@@ -788,24 +794,28 @@ private void writeEnumEAMethod(PrintWriter writer, String returnType, StructureD
 		}
 		writeMethodClose(writer);
 	}
+	
+	private String getRealOffsetName(FieldDescriptor fieldDescriptor) {
```

Comment:
The overall change could be much smaller if this new method is called `getOffsetConstant` and the old method of that name is changed to receive a string (the field name). None of the call sites in this file would then need to change.